css/src/components/form--select.css

```diff
@@ -0,0 +1,84 @@
+/**
+ * @file
+ * Select input elements.
+ */
+
+/**
+ * Limits extra long instances of select elements to the max width allowed
+ * to avoid breaking layouts.
+ */
+select {
```

Comment:
Since we started with a legacy project I think it makes sense to keep code what we didn't touched.

My idea was in the current case that the `form.css` is for styling the common elements (`form`, `form-item`, the labels, the descriptions etc).

I just moved the existing `select`-specific theming in a standalone file like I did for the _text_ elements.